lib/msal-browser/src/interaction_client/SilentAuthCodeClient.ts

```diff
@@ -83,7 +83,7 @@ export class SilentAuthCodeClient extends StandardInteractionClient {
             BrowserPerformanceEvents.StandardInteractionClientInitializeAuthorizationRequest,
             this.logger,
             this.performanceClient,
-            request.correlationId
+            this.correlationId
```

Comment:
The change from request.correlationId to this.correlationId occurs a few times in this file.  A comment on why one vs the other would be good for these as well.

Reply:
`correlationId` is optional in request payload, while `this.correlationId` is always instantiated as `request.correlationId || createGuid()`. Each auth request creates a new instance of `*Client` so we can safely use `this.correlationId`.

Reply:
Can you add a comment to the code so this is captured in there for future folks.